### ISSUE TITLE
Export full size tiff files from the digital collections to a directory on the P drive

### ISSUE THREAD

**Issue by eddierubeiz** [sciencehistory/scihist_digicoll#1414]
Download full size tiffs of works that are:
- in the "Papers of James Curtis Booth" collection
- have subject heading "Mint of the United States"
- ALSO have subject heading "United States Mint"

**Source:** the s3 originals bucket
**Destination:** `P:\Othmer Library\Archives\External Access\James Curtis Booth Images`

**Comment by apinkney0696** [comment]
Additionally, if you're able to keep the filenames as they exist in the collections that would be great. Otherwise, this all looks correct to me!

**Comment by eddierubeiz** [comment]
Yup. We have that file metadata. See e.g. https://digital.sciencehistory.org/admin/asset_files/2m79gmp .

**Comment by eddierubeiz** [comment]
See above -- this is going to the P drive.

**Comment by jrochkind** [comment]
Are those safe assumptions?  

The creating a wget bash script is a clever solution!

To avoid those assumptions, I guess you can iterate through any children recurisvely (yours would already error if a member was a Work, probably?), and get the S3 signed URL, whihc I think you can get from `mem.file.url` maybe?   In general actually, might as well get it right from S3 with a signed url instead of going through our app?

Curious how long it will take to download to dev computer though it could be way too long. 

This is definitely a good one to do from a 315 Chestnut location. In my home computer, I'd be downloading them to Baltimore then uploading them over my very slow comcast upload pipe to the P drive, I think it wouldn't work.
